mkosi.md

```diff
@@ -338,7 +338,7 @@ a boolean argument: either "1", "yes", or "true" to enable, or "0",
 `UseHostRepositories=`, `--use-host-repositories`
 
 : This option is only applicable for dnf-based distributions:
-  *CentOS*, *Fedora Linux*, *Mageia*, *Photon*, and *OpenMandriva*.
+  *CentOS*, *Fedora Linux*, *Mageia*, *Photon*, *Rocky*, and *OpenMandriva*.
```

Comment:
This might be a silly nibble, but what is the distribution's proper name? Grepping through the starting page of their website I only see "Rocky Linux" written out, not simply "Rocky". Is it maybe the same as with Fedora, that is properly called "Fedora Linux" (which we recently fixed all over the place)? Just want to make sure we don't have to fix this next. :)

Reply:
sure no probs, don't worry its not picky, its important to be correct :)